targets/targets.json

```diff
@@ -534,10 +534,21 @@
         "is_disk_virtual": true,
         "inherits": ["Target"],
         "detect_code": ["0218"],
-        "device_has": ["ANALOGIN", "ANALOGOUT", "ERROR_RED", "I2C", "I2CSLAVE", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "RTC", "SEMIHOST", "SERIAL", "SLEEP", "SPI", "SPISLAVE", "STDIO_MESSAGES"],
```

Comment:
are we having 2 KL82Z boards, one one K82Z ? Why that duplication in the case of KL82Z ? There could be one MCU target , the 2 targets would inherit the all definitions from its parent?